src/containers/old-dashboard/index.tsx

```diff
@@ -59,15 +104,15 @@ class OldDashboard extends React.Component {
       return;
     }
 
-    const startWidgetIds = Array.from(startSection.widgetIds);
-    startWidgetIds.splice(source.index, 1);
-    const newStartWidgetIds = {
-      ...startSection,
-      widgetIds: startWidgetIds,
-    };
-
+    // const startWidgetIds = Array.from(startSection.widgetIds);
```

Comment:
If you need this for the future then it's fine but if it's just old code, please remove.